sympy/calculus/singularities.py

```diff
@@ -189,6 +297,13 @@ def is_strictly_decreasing(expression, interval=S.Reals, symbol=None):
     >>> is_strictly_decreasing(-x**2 + y, Interval(-oo, 0), x)
     False
 
+    Returns
+    =======
+
+    Boolean
+        True if ``expression`` is strictly decreasing in the given ``interval``,
+        False otherwise.
```

Comment:
Examples should come after "Returns". The first two sections should be "Parameters" and "Returns". I think the numpydoc spec gives the best ordering of sections.